assets/js/modules/analytics-4/components/dashboard/AudienceSegmentation/AudienceSegmentationErrorWidget.stories.js

```diff
@@ -0,0 +1,164 @@
+}
+
+function Template( { setupRegistry = async () => {}, ...args } ) {
+	const setupRegistryCallback = async ( registry ) => {
+		provideModules( registry, [
+			{
+				active: true,
+				connected: true,
+				slug: 'analytics-4',
+			},
+		] );
+		provideModuleRegistrations( registry );
+		provideUserInfo( registry );
+		await setupRegistry( registry );
+	};
+	return (
+		<WithRegistrySetup func={ setupRegistryCallback }>
+			<AudienceErrorWidgetWrapper { ...args } />
+		</WithRegistrySetup>
+	);
+}
+
+export const Default = Template.bind( {} );
+Default.storyName = 'Default';
+Default.args = {
+	setupRegistry: async ( registry ) => {
+		await registry.dispatch( MODULES_ANALYTICS_4 ).receiveError(
+			{
+				code: 'test-error-code',
+				message: 'Test error message',
+				data: {
+					reason: 'Data Error',
+				},
+			},
+			'getReport',
+			[
+				{
+					dimensions: [ 'ga:date' ],
+					metrics: [ { expression: 'ga:users' } ],
+					startDate: '2020-08-11',
+					endDate: '2020-09-07',
+				},
+			]
+		);
+	},
+};
+Default.scenario = {
+	label: 'Modules/Analytics4/Components/AudienceSegmentation/AudienceSegmentationErrorWidget/Default',
+};
+
+export const InsufficientPermissions = Template.bind( {} );
+InsufficientPermissions.storyName = 'InsufficientPermissions';
+InsufficientPermissions.args = {
+	setupRegistry: async ( registry ) => {
+		const [ accountID, propertyID, measurementID, webDataStreamID ] = [
+			'12345',
+			'34567',
+			'56789',
+			'78901',
+		];
+
+		await registry
+			.dispatch( MODULES_ANALYTICS_4 )
+			.setAccountID( accountID );
+		await registry
+			.dispatch( MODULES_ANALYTICS_4 )
+			.setPropertyID( propertyID );
+		await registry
+			.dispatch( MODULES_ANALYTICS_4 )
+			.setMeasurementID( measurementID );
+		await registry
+			.dispatch( MODULES_ANALYTICS_4 )
+			.setWebDataStreamID( webDataStreamID );
+		await registry.dispatch( MODULES_ANALYTICS_4 ).receiveError(
+			{
+				code: 'test-error-code',
+				message: 'Test error message',
+				data: {
+					reason: ERROR_REASON_INSUFFICIENT_PERMISSIONS,
+				},
+			},
+			'getAccountID'
+		);
+	},
+};
+InsufficientPermissions.scenario = {
+	label: 'Modules/Analytics4/Components/AudienceSegmentation/AudienceSegmentationErrorWidget/InsufficientPermissions',
+};
+
+export default {
+	title: 'Modules/Analytics4/Components/AudienceSegmentation/AudienceSegmentationErrorWidget',
+	decorators: [
+		( Story ) => {
+			const registry = createTestRegistry();
+
+			return (
+				<div
+					style={ {
+						minHeight: '200px',
+						display: 'flex',
+						alignItems: 'center',
+					} }
+				>
+					<div id="adminmenu">
+						{ /* eslint-disable-next-line jsx-a11y/anchor-has-content */ }
+						<a href="http://test.test/?page=googlesitekit-settings" />
+					</div>
+					<div style={ { flex: 1 } }>
+						<WithTestRegistry registry={ registry }>
+							<Story />
+						</WithTestRegistry>
+					</div>
+				</div>
```

Comment:
I don't think this extra markup is needed is it? The placeholder admin menu is usually only needed for components which render `AdminMenuTooltip` so the tooltip has something to point to...

```suggestion
				<WithTestRegistry registry={ registry }>
					<Story />
				</WithTestRegistry>
```

Reply:
Updated ✅